src/citrine/informatics/predictor_evaluator.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional, Set, Type
+
+from citrine._serialization import properties
+from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
+from citrine._serialization.serializable import Serializable
+from citrine.informatics.predictor_evaluation_metrics import PredictorEvaluationMetric
+
+
+class PredictorEvaluator(PolymorphicSerializable["PredictorEvaluator"]):
```

Comment:
Is there a way to enforce that all sub-classes have a `name` field? I ask because that `name` will be used to get the execution's responses

Reply:
I'll add an `@property name -> NotImplementedError` method to `PredictorEvaluator`. I think that combined with a descriptive docstring might be close to what you're after. Ideally we'd use an abstract base class or define a Protocol. However the former breaks our serialization logic, and the latter isn't available in all the versions of Python we need to support.